src/devtools/client/webconsole/webconsole-wrapper.js

```diff
@@ -54,6 +73,12 @@ class WebConsoleWrapper {
     this.queuedRequestUpdates = [];
     this.throttledDispatchPromise = null;
     this.telemetry = new Telemetry();
+
+    ThreadFront.findConsoleMessages(this.onConsoleMessage);
+
+    LogpointHandlers.onPointLoading = this.onLogpointLoading;
```

Comment:
do these callbacks need to be `bind()`?

Reply:
nope because we're using class properties here (https://babeljs.io/docs/en/babel-plugin-proposal-class-properties)